examples/logrotate.cron

```diff
@@ -1,8 +1,23 @@
-#!/bin/sh
+#!/bin/bash
 
-/usr/sbin/logrotate /etc/logrotate.conf
-EXITVALUE=$?
-if [ $EXITVALUE != 0 ]; then
-    /usr/bin/logger -t logrotate "ALERT exited abnormally with [$EXITVALUE]"
+# exit immediately if there is another instance running
+if pgrep /usr/sbin/logrotate > /dev/null ; then
+	/bin/logger -p cron.warning -t logrotate "ALERT another instance of logrotate is running - exiting"
+	exit 1
 fi
```

Comment:
also, you changed path to `logger` binary in this commit. perhaps just rely on `$PATH` instead of hardcoding tool paths?